packages/nextjs/app/builders/0x119d9A1ef0D16361284a9661727b363B04B5B0c8/page.tsx

```diff
@@ -0,0 +1,175 @@
+                    alt="Profile image"
+                    className="object-cover"
+                    fill
+                    sizes="(max-width: 768px) 100vw, 192px"
+                  />
+                </div>
+              </div>
+
+              {/* Experience button moved beneath the profile image */}
+              <div className="relative mt-2">
+                <div className="absolute inset-0 border-4 border-purple-600 rounded-full transform rotate-3"></div>
+                <div className="relative border-2 border-purple-600 rounded-full px-8 py-3 inline-block bg-white">
+                  <div className="text-md text-center">Experience</div>
+                  <div className="text-xl font-bold text-purple-600 text-center">+1 YEARS</div>
+                </div>
+              </div>
+            </div>
+
+            {/* Bio content in the second and third columns */}
+            <div className="col-span-2">
+              <h2 className="text-4xl font-bold mb-4 uppercase">Dmystical-Coder</h2>
+
+              <div className="mb-6">
+                <p className="mb-4 text-lg">
+                  Hey folks, I&apos;m a passionate blockchain developer who loves to learn, grow and create awesome
+                  dApps. Yes I do create!! :)
+                </p>
+
+                <p className="mb-4">
+                  A passionate blockchain developer with a deep fascination for decentralized technologies. I specialize
+                  in creating intuitive dApps that bridge the gap between complex blockchain architecture and seamless
+                  user experience.
+                </p>
+              </div>
+
+              <div className="flex flex-wrap gap-4">
+                <div className="address-container border-2 border-[#385182] p-2 bg-white mb-6 text-sm break-all">
+                  <div className="font-bold uppercase mb-1">ETH Address:</div>
+                  <Address address={myAddress} format="long" />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        {/* Skills grid */}
+        <div className="border-4 border-[#385182] mb-8 bg-white">
+          <div style={{ backgroundColor: "#385182" }} className="text-white p-4">
+            <h2 className="text-xl font-bold uppercase tracking-wider">SKILLS</h2>
+          </div>
+
+          <div className="grid grid-cols-1 md:grid-cols-2 border-t-4 border-[#385182]">
+            {/* Left column - Blockchain Skills */}
+            <div className="border-r-4 border-[#385182] p-6">
+              <h3 className="text-lg font-bold mb-4 uppercase">Blockchain</h3>
+
+              <ul className="space-y-2">
+                {blockchainSkills.map(skill => (
+                  <li key={skill} className="border-2 border-[#385182] p-3 hover:bg-yellow-100 transition-colors">
+                    <span className="font-bold">{skill}</span>
+                  </li>
```

Comment:
You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.